.github/workflows/release-github.yml

```diff
@@ -1,6 +1,9 @@
 name: Release to GitHub
 on:
```

Comment:
The `on` property is a hash that can contain any of GitHub's allowed [events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).

Prior to this change, only the [workflow_dispatch event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) would trigger a release. That means the job had to be manually run.

With this change, the `on` property now _also_ contains the [push event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows), which is raised when a check-in to the repo happens. Here, the `push` event is configured to only care about pushes (aka check-ins or commits) to the `next` branch.

## End Result

The release workflow can still be manually triggered. But also, any check-in to the `next` branch will trigger a github pre-release and an npm release to the `@next` release channel.

Reply:
awesome, thanks!